### DISCUSSION TITLE
./lotus/chain/vm/fvm.go:505   - cbor input had wrong number of fields

### DISCUSSION THREAD

**Discussion by lionsoul2014** [filecoin-project/lotus#10750] [Storage Provider]
i modified some codes in the storage/sealer and did not change anything about the chain impl, why the fvm will keep firing an error like this ?

i merge the v1.22.0 and v1.22.1 and it all works find.
but when i merge the v1.23.0 based on the v1.22.1 and then upgrade the binary and it keep firing errors like this.

**Comment by rjan90** [comment]
Hey @lionsoul2014!

Can you check your submodules (git submodule in /lotus), I suspect that the errors are coming from having the wrong submodules.

**Reply by lionsoul2014** [reply]
the filecoin-ffi ?

**Reply by lionsoul2014** [reply]
something like that, thanks for you reply!
